pkg/image/util/manifest.go

```diff
@@ -18,7 +17,7 @@ func GetImageManifestByIDFromRegistry(registry *url.URL, repositoryName, imageID
 	credentials := importer.NewBasicCredentials()
 	credentials.Add(registry, username, password)
 
-	repo, err := importer.NewContext(http.DefaultTransport, http.DefaultTransport).
+	repo, err := importer.NewContext().
```

Comment:
Is there a reason why this method doesn't accept a RepositoryRetriever interface?  That would prevent it having to initialize all of these things and make it more composable.